src/olympia/devhub/templates/devhub/api/key.html

```diff
@@ -45,24 +45,26 @@
         {% else %}
           <p>
           {% trans %}
-            You don't have any API credentials yet.
+            You don't have any API credentials.
           {% endtrans %}
           </p>
         {% endif %}
-        <p>
-          {% trans bug_link='https://github.com/mozilla/olympia/issues' %}
-            This feature is under active development. If you find a problem please <a target="_blank" href="{{ bug_link }}">report a bug</a>.
-          {% endtrans %}
-        </p>
       </fieldset>
       <div class="listing-footer">
-        <button id="generate-key" class="button prominent" type="submit">
+        <p class="footer-submit">
           {% if credentials %}
-            {{ _('Revoke and regenerate credentials') }}
+            <button id="revoke-key" class="button prominent" type="submit" name="action" value="revoke">
+              {{ _('Revoke')}}
```

Comment:
super nitty nit: it's nice to have a space after the closing parenthesis for readability